src/engine/sys/sdl_glimp.cpp

```diff
@@ -101,6 +101,13 @@ static Cvar::Cvar<bool> r_ext_texture_integer( "r_ext_texture_integer",
 static Cvar::Cvar<bool> r_ext_texture_rg( "r_ext_texture_rg",
 	"Use GL_EXT_texture_rg if available", Cvar::NONE, true );
 
+static Cvar::Cvar<bool> workaround_noTextureGather_nvidia340( "workaround.noTextureGather.nvidia340", "Disable ARB_texture_gather on Nvidia 340 driver", Cvar::NONE, true );
+static Cvar::Cvar<bool> workaround_firstProvokingVertex_intel( "workaround.firstProvokingVertex.intel",
+"Use first provoking vertex on Intel hardware supporting ARB_provoking_vertex", Cvar::NONE, true );
+static Cvar::Cvar<bool> workaround_noBindlessTexture_mesa241( "workaround.noBindlessTexture.mesa241", "Disable ARB_bindless_texture on Mesa 24.1 driver", Cvar::NONE, true );
```

Comment:
Why do we need 2 different cvars about enabling the same bindless texture feature?

Reply:
This is not about enabling a bindless texture feature, it's is about disabling a workaround preventing bindless texture usage in a special case. The idea is to have a `workaround` cvar for disabling each workaround use case. Those are debugging cvars and having fine granularity is better.